__tests__/app.test.js

```diff
@@ -82,4 +80,19 @@ describe('GET /api/articles/:article_id', () => {
             expect(body.msg).toBe("Invalid input")
         })
     })
+})
+describe('GET /api/articles', () => {
+    it('returns status 200 and list of all articles in test DB in descending order', () => {
+        return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then(({ body }) => {
+                const articleKeys = ['author', 'title', 'article_id', 'topic', 'created_at', 'votes', 'article_img_url', 'comment_count']
+                expect(body.articles).toBeSortedBy('created_at', { descending: true })
+                body.articles.forEach((article) => {
+                    expect(Object.keys(article)).toEqual(expect.not.arrayContaining(['body']))
+                    expect(Object.keys(article)).toEqual(expect.arrayContaining(articleKeys))
```

Comment:
We have jest matchers which allow us to check objects such as ObjectContaining and ToMatchObject.
These allow much more robust testing.
There is no need to use Object.keys to get array values from them.

Reply:
When I try using ObjectContaining or ToMatchObject my test fail because it matches the values, but I want to check for keys only